apps/judicial-system/web/src/routes/Shared/Cases/ActiveCases.tsx

```diff
@@ -138,9 +132,11 @@ const ActiveCases: React.FC<React.PropsWithChildren<Props>> = (props) => {
       {displayCases.map((theCase: CaseListEntry) => (
         <Box marginTop={2} key={theCase.id}>
           <MobileCase
-            onClick={() => handleOpenCase(theCase.id)}
+            onClick={() => {
+              handleOpenCase(theCase.id)
+            }}
             theCase={theCase}
-            isCourtRole={isDistrictCourtUser(user)}
+            isCourtRole={false}
```

Comment:
Is this correct? Are we not showing this table to court roles?

Reply:
After the current change `ActiveCases` is only used for prosecution roles.